beetsplug/alternatives.py

```diff
@@ -117,6 +117,12 @@ def parse_config(self, config):
             dir = os.path.join(self.lib.directory, dir)
         self.directory = dir
 
+        if not config['linktype'] in ['absolute', 'relative']:
```

Comment:
How about using `relativeLinks` as the key. Then you can use `config.get['relativeLinks'].as_bool()` (I think).

Reply:
https://confit.readthedocs.io/en/latest/#validation

You can not use as_bool, but I'll use `as_choice({"yes":True, "no":False})`

Reply:
Ah, yes. Does this also raise an error if the value is not `yes` or `no`?